packages/core/src/storage/__tests__/IndyStorageService.test.ts

```diff
@@ -113,6 +114,13 @@ describe('IndyStorageService', () => {
 
       expect(record).toEqual(found)
     })
+
+    it('After a save the record should have update the updatedAt property', async () => {
+      const time = new Date().getTime()
+      await sleep(1000)
+      const record = await insertRecord({ id: 'test-updatedAt' })
+      expect(record.updatedAt?.getTime()).toBeGreaterThan(time)
+    })
```

Comment:
In general it's better to mock the date, as this will make the test wait for a second without specific reason which increases the overall time it takes to run tests

https://dev.to/maxpou/how-to-mock-date-with-jest-3k4b

Reply:
Instead of mocking the dates I just got the time before all the tests had started and made sure that after an update and save the updated at time was after that initial time.